eland/_version.py

```diff
@@ -12,9 +12,9 @@
 #      See the License for the specific language governing permissions and
 #      limitations under the License.
 
-__title__ = 'eland'
-__description__ = 'Python elasticsearch client to analyse, explore and manipulate data that resides in elasticsearch.'
-__url__ = 'https://github.com/elastic/eland'
-__version__ = '7.6.0a4'
-__maintainer__ = 'Elasticsearch B.V.'
-__maintainer_email__ = 'steve.dodson@elastic.co'
+__title__ = "eland"
+__description__ = "Python elasticsearch client to analyse, explore and manipulate data that resides in elasticsearch."
+__url__ = "https://github.com/elastic/eland"
+__version__ = "7.6.0a4"
+__maintainer__ = "Elasticsearch B.V."
+__maintainer_email__ = "steve.dodson@elastic.co"
```

Comment:
I wonder if we should change the maintainer email to something that's not tied to a single person's email account

Reply:
Could do that, also was wondering if we should move Steve into the author category and maintainer can be the group email / someone else's email.

Reply:
@sethmlarson - happy to be moved out. You should be the primary 'owner' now.